tests/integration/supplychain/supply_chain_suite_test.go

```diff
@@ -65,23 +65,7 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 
 	// start kube-apiserver and etcd
-	testEnv = &envtest.Environment{
-		WebhookInstallOptions: envtest.WebhookInstallOptions{
-			Paths: []string{filepath.Join("..", "..", "..", "config", "webhook")},
-		},
-		CRDDirectoryPaths: []string{
-			filepath.Join("..", "..", "..", "config", "crd", "bases"),
-			filepath.Join("..", "..", "resources", "crds"),
-		},
-
-		AttachControlPlaneOutput: DebugControlPlane, // Set to true for great debug logging
-	}
-
-	if DebugControlPlane {
-		testEnv.ControlPlane.GetAPIServer().Configure().
-			Append("audit-policy-file", filepath.Join(workingDir, "policy.yaml")).
-			Append("audit-log-path", "-")
-	}
+	testEnv = integration.CreateTestEnv(workingDir, GinkgoWriter)
```

Comment:
The `const DebugControlPlane = false` 8 lines prior looks like it should be removed.